src/app/home/home.component.ts

```diff
@@ -1,7 +1,9 @@
 import { Component, OnInit } from '@angular/core';
 import { finalize } from 'rxjs/operators';
-import { IWeather, IWeatherCurrent, IForecastDay, IDay } from '../shared/interfaces/weather.interface'
+import { IWeather, IWeatherCurrent, IForecastDay, IDay } from '../shared/interfaces/weather.interface';
+import { IError } from '../shared/interfaces/error.interface';
 import { WeatherService } from '@app/shared/weather.service';
+import { MatSnackBar } from '@angular/material/snack-bar';
```

Comment:
It looks nice, buuuuuuuut (there's always a but :stuck_out_tongue: ), I really want to see this component implemented by yoursef in the last project task haha :muscle:

Reply:
hahahahah, that's will be funny, right? I'll do it!